src/pages/MyPatches.tsx

```diff
@@ -38,45 +38,26 @@ const MyPatchesComponent: React.FC = () => {
 
   const { replace } = useHistory();
   const { search, pathname } = useLocation();
-  const [initialQueryVariables] = useState<UserPatchesQueryVariables>({
-    page: 0,
-    ...getQueryVariables(search),
-  });
+  const [initialQueryVariables] = useState<UserPatchesQueryVariables>(
+    getQueryVariables(search)
+  );
   const [
     patchNameFilterValue,
     patchNameFilterValueOnChange,
   ] = useFilterInputChangeHandler(MyPatchesQueryParams.PatchName);
-  const { data, fetchMore, loading, error } = useQuery<
+  const { data, refetch, networkStatus, error } = useQuery<
     UserPatchesQuery,
     UserPatchesQueryVariables
   >(GET_USER_PATCHES, {
     variables: initialQueryVariables,
     notifyOnNetworkStatusChange: true,
   });
-
-  useEffect(() => {
-    const fetch = async () => {
-      try {
-        await fetchMore({
-          variables: {
-            ...getQueryVariables(search),
-          },
-          updateQuery: (
-            prev: UserPatchesQuery,
-            { fetchMoreResult }: { fetchMoreResult: UserPatchesQuery }
-          ) => {
-            if (!fetchMoreResult) {
-              return prev;
-            }
-            return fetchMoreResult;
-          },
-        });
-      } catch (e) {
-        // empty block
-      }
-    };
-    fetch();
-  }, [search, fetchMore]);
+  const { showSkeleton } = usePollTableQuery({
```

Comment:
should the name of `usePollTableQuery` be changed since it's used to poll things that are not a table, such as the list of user patches?

Reply:
> Did you file a ticket with Leafygreen to address the performance issue with their tooltip?

the Leafygreen lead engineer is aware of the issue